canary/generator/download/download_broke.py

```diff
@@ -0,0 +1,68 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
nit: there's a typo in the filename, but let's rename this to "download_split_data.py"